### PR TITLE
Fix crash when xkb keyboard is not used.

### PR DESCRIPTION
**Pull request by msisov**
This is a simple fix, which fixes usage of non-xkb keyboards. Ideally, we should make a keyboard engine
to support other keyboards.

### CODE REVIEW

**Comment by msisov** [comment]
gentle reminder

**Comment by tonikitoo** [comment]
quick question: does keyboard input work?

**Comment by msisov** [comment]
yes, it does, but without modifiers (ctrl+click, for example) and etc. Just basic input

**Comment by tonikitoo** [comment]
Ok, although maybe we should not crash (DCHECK) as we do today, maybe we want to warn the use somehow once? Other than this, it looks ok to me.

"XKB is not available. Some keyboard modifiers will not be functional."

File: ui/ozone/platform/wayland/wayland_keyboard.cc [hunk 1]
```diff
@@ -63,8 +63,8 @@ void WaylandKeyboard::Keymap(void* data,
   bool success =
       KeyboardLayoutEngineManager::GetKeyboardLayoutEngine()
           ->SetCurrentLayoutFromBuffer(keymap_str, strnlen(keymap_str, size));
-  DCHECK(success) << "Failed to set the XKB keyboard mapping.";
-  munmap(keymap_str, size);
+  if (success)
+    munmap(keymap_str, size);
```
Comment (tonikitoo) [inline ui/ozone/platform/wayland/wayland_keyboard.cc]: sorry, forgot about this.

I wonder if would could do something like


````
...
if (!success) {
  LOG(ERROR) << "Failed to set the XKB keyboard mapping. Only basic keyboard support available.";
  return;
}

munmap(keymap_str, size);
}

Comment (msisov) [reply ui/ozone/platform/wayland/wayland_keyboard.cc]: Done

**Review by tonikitoo — APPROVED** [review]
lgtm

**Pull request merged**